Dockerfile

```diff
@@ -21,10 +21,14 @@ RUN set -x && \
     apt-get install -y ansible && \
     echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
 
+ENV DEBIAN_FRONTEND noninteractive
+
 # Install OpenCV 3.2
-RUN apt-get update -y && apt-get upgrade -y && apt-get autoremove -y && \
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get autoremove -y && \
     apt-get install -y --no-install-recommends cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev \ 
-    libswscale-dev python3.5-dev python3-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff5-dev libjasper-dev \
+    libswscale-dev python3.6-dev python3-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff5-dev jasper \
```

Comment:
How do you know this is the same as `libjasper-dev`?

Reply:
> How do you know this is the same as `libjasper-dev`?

@amverni I suppose I don't, but OpenCV installed successfully on the Jetson using this `apt` command with these dependencies and `libjasper-dev` is not available on Ubuntu 18.04